ip-messaging/rest/messages/delete-messages/delete-messages.5.x.php

```diff
@@ -5,15 +5,15 @@
 use Twilio\Rest\Client;
 
 // Find your Account Sid and Token at twilio.com/user/account
-$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
-$token = "your_auth_token";
+$sid = "ACCOUNT_SID";
```

Comment:
These placeholder values are conventional, and are replaces with live values during test and when displayed by the UI in the docs - can you back out these changes?